README.md

```diff
@@ -30,6 +30,34 @@ make demo
 The build script will then download data file for Doom automatically. SDL2 based window
 should appear when Doom is loaded and executed.
 
+## riscv-arch-test
+
+The RISC-V Architectural Tests, [riscv-arch-test](https://github.com/riscv-non-isa/riscv-arch-test) is basic set of tests that ensure software implemented can be match important aspects of the RISC-V specification.(not a **substitute for rigorous design verification**)
```

Comment:
Check the grammar: "ensure software implemented can be match important aspects of ..."